### DISCUSSION TITLE
Couple questions about app

### DISCUSSION THREAD

**Discussion by Nostang3** [deckerst/aves#177] [Q&A]
I've been searching for a new gallery app to replace my old one. It currently works but the developer abandoned it and eventually it won't. My current app is really great about being able to add tags to photos or filtered searches with tags.

1. I've seen discussions about it but I can't find a simple way to add an album to the application. I have my own directory called albums and within it an album I want to add. Maybe this is related to question 2.
2. Can you add said album if it has a .nomedia file in them? I want the album to be hidden to other apps so media share isn't flooded with thousands of photos but would like the option to add it to my gallery app with the option to toggle it's visibility within. 
3. I did some quick tag testing and it looks like when the app filters tags based on the AND operator. Is there a way or could there be a way to search for multiple tags search based on OR? Like a toggle option.
4. Followup on 2, if the album is hidden in the app do all of the tags used in that album become hidden as well?

**Comment by deckerst** [comment]
1. The collection you see in Aves is based on the Android native Media Store. Aves allows hiding folders (and other filters, like tags, locations, etc.) but it doesn't allow adding folders that are out of scope of the Media Store. This is related to 2.
2. See my answer [here](https://github.com/deckerst/aves/pull/169#issuecomment-1025713394). Aves is an app that endeavors to work *with* the system, not against it. If something is in a ".nomedia" folder, it is not media from the point of view of the system, and Aves respects this.
3. You're right: for now only AND is supported, but I see the value of having OR too. One thing I'd like to avoid is exposing the feature to the user as a geeky syntax that requires an explanation (like F-Stop does), so i need to think of a non-obtrusive way. Maybe in the context menu of the filters, not sure... Would you use multiple tags by applying them from the Search page?
4. If the tags were *only* used in hidden media, yes the tags will not be visible. By the way, you can hide a tag too (via a tag filter context menu).

**Reply by Nostang3** [reply]
First of all thanks for taking the time to answer the questions.

1 and 2. I can understand your design choice. I might have to change who I do things if I choose to use the app. 
3. You could use the words ANY or ALL. It's simple wording and not geeky syntax. I believe that would be the best way would it not? My use case is that I will tag a new photo with some tags, then search for all photos with those tags to see if there is duplicate photos with that tag. I have to do this because android duplicate photo finders are limited in scope. 
4. Hidden media in the app, not via the .nomedia file, right? I just want to make sure we are on the same page.

**Reply by deckerst** [reply]
4. Yes, hidden in the app.
